mkosi/user.py

```diff
@@ -39,7 +39,7 @@ def is_running_user(cls) -> bool:
     @classmethod
     @functools.lru_cache(maxsize=1)
     def name(cls) -> str:
-        return pwd.getpwuid(cls.uid).pw_name
+        return os.getenv("USER", pwd.getpwuid(cls.uid).pw_name)
```

Comment:
I think asking first and falling back to the env would be better, but I'm fine either way.
```suggestion
        try:
            return pwd.getpwuid(cls.uid).pw_name
        except KeyError:
            if name := os.getenv("USER"):
                return name
            raise
```

Reply:
I think it's more consistent to let environment win like we do elsewhere